### ISSUE TITLE
Feature Request: Use of the median instead of the mean to aggregate the individual tree predictions.

### ISSUE THREAD

**Issue by nikosGeography** [imbs-hl/ranger#740]
"Classical" random forest uses the mean to aggregate the individual tree predictions. I was wondering if it make sense to provide the user with alternative methods, such as the median, for the aggregation. For example, a paper that utilizes the median is: "Robustness of random forests for regression". Do you think this is something that will benefit the users of the ranger package? If not, feel free to close the issue.

Many thanks.

**Comment by mnwright** [comment]
Yes, that's a good idea and shouldn't be very hard to implement.
